### DISCUSSION TITLE
Adding Family members

### DISCUSSION THREAD

**Discussion by Sp0run** [liedekef/events-made-easy#165] [General]
Is there an easy way to allow users to add their family members to a current membership? 

The scenario is that I'm running a club which charges for annual membership, we have a few people who want to sign their children up since we started youth coaching and I want to make this easy to manage for them so that they only have to pay one subscription charge.

I haven't anything in the documentation so I'm assuming there isn't. 
If I need to code this myself any help or thoughts on how to achieve it would be gratefully received.

**Comment by liedekef** [comment]
Once a member, people can't change their member info themselves anymore. That's only possible in the eme backend.
